api/apps/articles/model/schema.coffee

```diff
@@ -30,15 +30,16 @@ videoSection = (->
     background_color: @string().allow('',null)
 ).call Joi
 
-fullscreenSection = (->
+featureSection = (->
   @object().meta(
-    name: 'Fullscreen'
-    isTypeOf: (data) -> data.type is 'fullscreen'
+    name: 'FeatureHeader'
+    isTypeOf: (data) -> data.type in ['fullscreen', 'split', 'text']
   ).keys
-    type: @string().valid('fullscreen')
+    type: @string().valid('fullscreen', 'split', 'text')
     title: @string().allow('',null)
-    intro: @string().allow('',null)
-    url: @string().allow(null)
+    intro: @string().allow('',null) # TODO - Remove after backfill
+    deck: @string().allow('',null)
+    url: @string().allow('',null)
```

Comment:
Note that we'll also need to update the GraphQL query here: https://github.com/artsy/force/blob/master/desktop/apps/article2/queries/articleBody.js#L39

Reply:
More like "note to self" -- I have a PR coming up with some updates so I'm happy to include there.

Reply:
Oh good to know, am happy to do a quick PR or if you're able to include that is great too -- I guess we can't deploy to production until that change is in.

Reply:
Sure go ahead! Should just be that simple swap.